### PR TITLE
create tabuada

### PR DESCRIPTION
**Pull request by FilipeJMachado**
mumu ve se ta certo esse progaminha

### CODE REVIEW

File: tabuada.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+number = int(input("Digite o numero que deseja a tabuada: "))
+multiplo = 0
```
Comment (MuriloFazio) [inline tabuada.py]: typo

Comment (FilipeJMachado) [reply tabuada.py]: typo zero ?

Comment (MuriloFazio) [reply tabuada.py]: change ''multiplo'' variable to an english word

Comment (FilipeJMachado) [reply tabuada.py]: revisa de novo porra

Comment (FilipeJMachado) [reply tabuada.py]: brincadeira s2

File: tabuada.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+number = int(input("Digite o numero que deseja a tabuada: "))
+multiple = 0 
+print(f"TABUADA DE {number}")
+for i in range(1,11): 
+    multiple += 1 
+    print(f"{number} x {multiplo} = {number * i}")
```
Comment (MuriloFazio) [inline tabuada.py]: code is broaken, multplo variable doesn´t exist

**Review by MuriloFazio — APPROVED** [review]

**Pull request merged**